lib/order.rb

```diff
@@ -0,0 +1,91 @@
+require_relative 'customer'
+
+class Order
+	attr_reader :id
+	attr_accessor :products, :customer, :fulfillment_status
+
+	def initialize(id, products, customer, fulfillment_status = :pending)
+		@id = id
+		@products = products
+		@customer = customer
+		self.fulfillment_status = fulfillment_status
+	end
+
+	# Raises argument errors if status is not recognized
+	def fulfillment_status=(value)
+		potential_status = [:pending, :paid, :processing, :shipped, :complete]
+		if potential_status.include? value
+			@fulfillment_status=(value)
+		else
+			raise ArgumentError,"Status is bogus"
```

Comment:
Generally speaking it's helpful to say what value was invalid for an argument for ease of debugging and clarity to the user.

```suggestion
			raise ArgumentError, "Status #{status} is bogus"
```